tests/test_numpy_array.cpp

```diff
@@ -245,39 +248,59 @@ TEST_SUBMODULE(numpy_array, sm) {
     });
 
     // test_overload_resolution
+    // NOLINTNEXTLINE(performance-unnecessary-value-param)
```

Comment:
These all are nameless but could all be const ref. I didn't know if that would break the overloads though.